### ISSUE TITLE
[Typescript] Exported abstract class looses `abstract` highlight

### ISSUE THREAD

**Issue by rchl** [sublimehq/Packages#3060]
## Code

```ts
abstract class A {
}

export abstract class B {
}
```

## Screenshot

![Screenshot 2021-09-30 at 12 28 08](https://user-images.githubusercontent.com/153197/135439010-6293a7e3-9ee4-4dd2-b623-64c913de03c7.png)

**Comment by rchl** [comment]
Also:

```
export declare enum Type {
    A = "A",
    B = "B",
}
```

vs.

```
declare enum Type {
    A = "A",
    B = "B",
}
```

is missing storage type for the `declare` in the former case

**Comment by Thom1729** [comment]
Thanks. Should be fixed in #3061.
